src/app/cluster/details/shared/version-picker/style.scss

```diff
@@ -41,20 +41,33 @@
 
         margin: 0 auto;
       }
+    }
 
-      .km-icon-update-available,
-      .km-icon-warning,
-      .km-icon-error {
-        @include mixins.size(16px);
+    .update-available-icon {
+      @include mixins.size(16px);
 
-        margin: 0 0 12px 22px;
-        position: absolute;
-      }
+      position: absolute;
+      right: 28px;
+      top: -9px;
+    }
+
+    .warning-icon,
+    .error-icon {
+      @include mixins.size(16px);
+
+      margin-right: 0;
+      position: absolute;
+      right: 28px;
+      top: -13px;
```

Comment:
One last thing is if we can style this without position absolute? It should be possible as right now matSuffix is used.

Reply:
without position absolute the icon will always be placed on the right of the dropdown arrow :/